### PR TITLE
part4 firebase deploy

### PR DESCRIPTION
**Pull request by DreRandaci**

### CODE REVIEW

**Comment by DreRandaci** [comment]
https://angular-contacts-ff671.firebaseapp.com/

File: partials/contact-edit.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div id='edit_contact' class='text-center container'>
```
Comment (morecallan) [inline partials/contact-edit.html]: If this form is exactly the same as your newContact partial, you can reuse that...

Comment (DreRandaci) [reply partials/contact-edit.html]: meaning, not needing the edit partial all together?

Comment (morecallan) [reply partials/contact-edit.html]: Bingo

Comment (DreRandaci) [reply partials/contact-edit.html]: so, the directions specify that on the pencil icon click the user should be rerouted to an edit/:id page, how do i grab the html from the new contact page to display there?

Comment (morecallan) [reply partials/contact-edit.html]: Oh shoot, just now seeing this! - You can just have your templateURL point to your newContact partial (html).

**Comment by DreRandaci** [comment]
@morecallan done https://angular-contacts-ff671.firebaseapp.com

**Comment by morecallan** [comment]
👍

**Pull request merged**